This fixes int10 vbeSetDisplayStart
without this patch you get a black screen.
(tested successfully using geforce4, intel i915)

diff -Nur vesautils-20050918.orig/libvbe/vbe.c vesautils-20050918/libvbe/vbe.c
--- vesautils-20050918.orig/libvbe/vbe.c	Fri Aug 12 20:53:54 2005
+++ vesautils-20050918/libvbe/vbe.c	Fri May 26 16:30:34 2006
@@ -651,7 +651,7 @@
     pixel_num = offset%(unsigned long)curr_mode_info.BytesPerScanLine;
     if(pixel_num*(unsigned long)curr_mode_info.BytesPerScanLine!=offset) pixel_num++;
     r.eax = 0x4f07;
-    r.ebx = vsync ? 0x82 : 2;
+    r.ebx = vsync ? 0x80 : 0;
     r.ecx = pixel_num;
     r.edx = offset/(unsigned long)curr_mode_info.BytesPerScanLine;
     if(!VBE_LRMI_int(0x10,&r)) return VBE_VM86_FAIL;
